nrf_802154/serialization/src/nrf_802154_spinel_app.c

```diff
@@ -279,6 +279,49 @@ static nrf_802154_ser_err_t capabilities_await(uint32_t                    timeo
     return error;
 }
 
+/**
+ * @brief Wait with timeout for time property to be received.
+ *
+ * @param[in]  timeout  Timeout in us.
+ * @param[out] p_time   Pointer to the time variable which needs to be populated.
+ *
+ * @returns  zero on success or negative error value on failure.
+ *
+ */
+static nrf_802154_ser_err_t time_await(uint32_t   timeout,
+                                       uint32_t * p_time)
+{
+    nrf_802154_ser_err_t              res;
+    nrf_802154_spinel_notify_buff_t * p_notify_data = NULL;
+
+    SERIALIZATION_ERROR_INIT(error);
+
+    p_notify_data = nrf_802154_spinel_response_notifier_property_await(
+        timeout);
+
+    SERIALIZATION_ERROR_IF(p_notify_data == NULL,
+                           NRF_802154_SERIALIZATION_ERROR_RESPONSE_TIMEOUT,
```

Comment:
Question (curiosity): This code uses spaces not tabs. Intentional?

Reply:
The code inside the `nrf_802154` is formatted according to the Nordic Coding Standard (not Zephyr's), so - yes, intentional.